### PR TITLE
[ci] Return test exit status from Travis start scripts

### PR DESCRIPTION
**Pull request by ChrisBr**
otherwise the container exits successfully. This makes it work in Travis and locally.

Co-authored-by: Victor Pereira <vpereirabr@gmail.com>

### CODE REVIEW

**Comment by hennevogel** [comment]
All of this is for running the CI containers locally?

File: contrib/start_lint [hunk 1]
```diff
@@ -6,6 +6,8 @@ echo "Preparing application..."
 bundle exec rake dev:prepare RAILS_ENV=test
 echo "Running linter..."
 bundle exec rake dev:lint RAILS_ENV=test
+tmp_exit=$?
```
Comment (Ana06) [inline contrib/start_lint]: please no Perl variables! :sob: Use the equivalent `$CHILD_STATUS` from the [English library](https://ruby-doc.org/stdlib-2.5.1/libdoc/English/rdoc/English.html). You need to include the library.

Rubocop should also complain.


BTW, it seems that the Ruby core team finally plans to remove Perl variables in Ruby 3 :tada:

Comment (ChrisBr) [reply contrib/start_lint]: @Ana06 this is bash, not ruby :smile:

**Comment by ChrisBr** [comment]
> All of this is for running the CI containers locally?

I also think this is the correct implementation, we could remove the if CI basically now!

File: contrib/start_lint (hunk 1 above)
Comment (Ana06) [reply contrib/start_lint]: ouch... :see_no_evil:

**Comment by ChrisBr** [comment]
I removed the CI conditional because we don't need it anymore! Also fixed the rubocop issues.

**Pull request merged**